src/test/java/io/jmnarloch/spring/boot/rxjava/mvc/ObservableReturnValueHandlerTest.java

```diff
@@ -63,6 +69,20 @@
     @EnableAutoConfiguration
     @RestController
     protected static class Application {
+        private @Autowired RequestMappingHandlerAdapter adapter;
+
+        // Needed b/c ObservableJust in rxjava2 implements Callable.
+        // The Callable handler is the first eligible that is picked up.
+        @PostConstruct
+        public void prioritizeCustomReturnValueHandlers () {
```

Comment:
Hmm I wonder if don't need a better control of registration of the handlers, so this would work with out such hack.